app/routes/erddap/buoy.js

```diff
@@ -22,35 +22,47 @@ router.param("source", (req, res, next, source) => {
 
 /**
  * @swagger
- * /erddap/:source/query:
+ * /erddap/{source}/query:
  *   get:
  *     description: Get Data from ERDDAP
  *     parameters:
- *      - name: ids
- *        in: query
- *        description: Buoy IDs, comma separated
- *        required: true
- *        type: string
- *      - name: variables
- *        in: query
- *        description: Measurement Variables, comma separated
- *        required: true
- *        type: string
- *      - name: start
- *        in: query
- *        description: Start Time
- *        required: true
- *        type: string
- *      - name: end
- *        in: query
- *        description: End Time
- *        required: true
- *        type: string
- *      - name: numPoints
- *        in: query
- *        description: max number of points to return
- *        required: false
- *        type: integer
+ *       - in: path
```

Comment:
So swagger works? ;)

Reply:
yup - turns out it's been there all along 🙃